### ISSUE TITLE
A types.d.ts file for integrated code editors.

### ISSUE THREAD

**Issue by AqilCont** [alpacahq/alpaca-trade-api-js#66]
Is it possible to include a types.d.ts file in the project? It can help people working with this API a lot, especially people using code editors that support it(like me, using VSCode). I can even help you a bit if given a reply.

**Comment by shlomikushchi** [comment]
you should refer to: https://github.com/alpacahq/alpaca-trade-api-js/issues/9#issuecomment-636642202

**Comment by AqilCont** [comment]
I am physically working on it with him, but it would help if there was also one in the official API linked on the website.

**Comment by poteat** [comment]
Yep - although 117's project is great, I think people would be more comfortable with a typing definitions file for the official API wrapper - using 117's forked project is overkill when you only want typing help.

**Comment by noramehesz** [comment]
Related [issue]( https://github.com/alpacahq/alpaca-trade-api-js/issues/96). I will close this one.
